.teamcity/pom.xml

```diff
@@ -118,6 +118,27 @@
                     </execution>
                 </executions>
             </plugin>
+            <plugin>
+                <groupId>org.codehaus.mojo</groupId>
+                <artifactId>exec-maven-plugin</artifactId>
+                <version>3.0.0</version>
+                <executions>
+                    <execution>
+                        <id>update-test-buckets</id>
+                        <phase>install</phase>
+                        <goals>
+                            <goal>java</goal>
+                        </goals>
+                    </execution>
+                </executions>
+                <configuration>
+                    <mainClass>model.FunctionalTestBucketGeneratorKt</mainClass>
+                    <arguments>
+                        <argument>./test-class-data.json</argument>
+                        <argument>./test-buckets.json</argument>
```

Comment:
I thought you wanted to use a system property for this location.

Reply:
Used system property.

Reply:
I think this is a misunderstanding: So passing the locations to the main class as command line arguments is fine. What I would do is to create a Maven property in the `pom.xml` with the default location and then use this here as arguments. So you can select a different location by using system properties on the command line when running Maven, while still have the default properties available.

Though it is also fine as it is now.